keyboards/trainpad/readme.md

```diff
@@ -0,0 +1,25 @@
+# trainpad
+
+Custom four key keyboard. **See each individual board for pin information.**
+
+* Keyboard Maintainer: [Ananya Kirti](https://github.com/AnanyaKirti)
+* Hardware Supported: custom stacked FR4 case
+* Hardware Availability: Contact bob at [IMKC Discord](https://discord.gg/imkc)
+
+
+
+## Bootloader
+
+Enter the bootloader:
+* **Physical reset button**: Briefly press the taticle button on the back of the PCB.
```

Comment:
You have Bootmagic enabled, so there's another possible reset method. Also fixes a typo.

```suggestion
* **Bootmagic reset**: Hold down the far left key while connecting the USB cable.
* **Physical reset button**: Briefly press the tactile button on the back of the PCB.
```